lua/lspconfig/server_configurations/eslint.lua

```diff
@@ -107,6 +107,13 @@ return {
         uri = new_root_dir,
         name = vim.fn.fnamemodify(new_root_dir, ':t'),
       }
+
+      -- Support Yarn2 (PnP) projects
+      local pnp_cjs = util.path.join(new_root_dir, '.pnp.cjs')
+      local pnp_js = util.path.join(new_root_dir, '.pnp.js')
```

Comment:
Globbing might be nicer here, but wanted to avoid `vim.fn` call.

Reply:
btw, this is not working with experimental `.pnp.loader.mjs` which is tries to load vscode-eslint-language-server esm format, and fails because of cjs module

Reply:
Yes it would help, but  because of this [issue](https://github.com/yarnpkg/berry/issues/3900) it doesn't work, but its not related to this PR